src/de/jost_net/JVerein/gui/view/EinstellungenSpendenbescheinigungenView.java

```diff
@@ -59,6 +59,7 @@ public void bind() throws Exception
     cont.addLabelPair("Unterschrift drucken",
         control.getUnterschriftdrucken());
     cont.addLabelPair("Unterschrift", control.getUnterschrift());
+    cont.addLabelPair("Anhang bei E-Mail Versand in DB speichern", control.getAnhangSpeichern());
```

Comment:
Der Eintrag steht zwar bei den Spendenquittungen, aber vielleicht wäre es trotzdem gut, hier explizit die Spendenquittung zu erwähnen, dass der Benutzer nicht verunsichert wird. Irgendwas wie "Spendenbescheinigung bei Mailversand in DB speichern" oder so?
Keine Ahnung, ob das unbedingt notwendig ist, ich fände es aber klarer für den Benutzer...

Reply:
Danke! Ist meiner Meinung nach jetzt klarer.